### ISSUE TITLE
Hidden sidebar causing other plugins issues

### ISSUE THREAD

**Issue by markbattistella** [docsifyjs/docsify#1392]
#### Steps to reproduce

1. Setup fresh site
1. Add config:
    ```js
    window.$docsify = {
      name: '',
      repo: '',

      // cover page doc
      coverpage: true,
      homepage: 'home.md',

      // remove sidebar
      hideSidebar: true,

      // auto scroll to top
      auto2top: true
    }
    ```
1. Create two or more `md` pages:
    1. `home.md`  -- add 2x `href` to  `page1` and `page2`
    1. `page1.md`  -- add 2x `href` to  `home` and `page2`
    1. `page2.md`  -- add 2x `href` to  `home` and `page1`
1. Launch http://localhost:3000
1. From `/#/home` in console no errors
    1. Click on `page1` link
    1. Continue to cycle through pages without hard refresh

#### What is current behaviour

After first load and the sidebar is removed, any other navigation to other pages triggers the `.remove` which fails since the element is already gone.

If the error is hit, then following plugins dont run until a hard reload.

```js
Uncaught TypeError: Cannot read property 'remove' of null
    at Vn.Xn._renderSidebar (docsify.min.js:1)
    at Vn.Pn (docsify.min.js:1)
    at docsify.min.js:1
    at o (docsify.min.js:1)
    at B (docsify.min.js:1)
    at t (docsify.min.js:1)
    at docsify.min.js:1
    at docsify.min.js:1
    at docsify.min.js:1
    at On (docsify.min.js:1)
```

```js
Line 4150: return document.querySelector("aside.sidebar").remove(),
```

#### What is the expected behaviour

Error doesn't occur, and doesn't stop other scripts from running.

#### Other relevant information

- [x] Bug does still occur when all/other plugins are disabled?

- Your OS: macOS
- Node.js version: 14.8.0
- npm/yarn version: 6.14.8
- Browser version: 
- Docsify version: 4.11.6
- Docsify plugins:

**Comment by sy-records** [comment]
If you have a `hideSidebar`, why do you need to operate the sidebar?

**Comment by anikethsaha** [comment]
Yes, I was not 100% comfortable with styling like these. so marked as fixme inorder to re-iterate here again.

**Comment by markbattistella** [comment]
@sy-records I was making a single page layout with hard coded navigation and noticed that other plugins weren't loading. Everything was stopping after the `.remove` was failing since the sidebar was already hidden and removed

**Comment by jhildenbiddle** [comment]
This issue has been addressed by PR #2469. The changes will be available when Docsify v5 is released.
